tests/variables/list-unpacking.du

```diff
@@ -9,6 +9,20 @@ class TestListUnpacking < UnitTest {
     testListUnpacking() {
         var [a, b, c] = [1, 2, 3];
 
+        assert(a == 1);
+        assert(b == 2);
+        assert(c == 3);
+
+        var o,i,g;
+
+        [o,i,g]=[111,222,333];
+
+        assert(o == 111);
+        assert(i == 222);
+        assert(g == 333);
+
+        var [d, e, f] = [1 + 1, 2 + 2, 3 + 3];
+
```

Comment:
We want to use the https://dictu-lang.com/docs/standard-lib/unittest/ builtin where possible over `assert` as `assert` will stop execution on first failure (as the main reason - UnitTest library has a nicer output too).